recipes/lsst-ts-xml/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "lsst-ts-xml" %}
+{% set version = "20.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/lsst-ts/ts_xml/archive/v{{ version }}.tar.gz
+  sha256: 04a8c8053c1c26180b689c61ab470de8117f6ac243158f17b17aa1a288dfc5fe
+
+build:
+  skip: True  # [win, py<310]
```

Comment:
This recipe should be noarch python with a constraint on python in the recipe directly.

Reply:
Except it didn't build successfully with Windows and therefore when I set the skip: true [win], the linter said that having the skip meant that noarch:python couldn't be used.

Reply:
You can ignore the windows failure. Noarch python only builds on Linux.